consensus/consensus_v2.go

```diff
@@ -103,7 +103,12 @@ func (consensus *Consensus) announce(block *types.Block) {
 
 	consensus.block = encodedBlock
 	consensus.blockHeader = encodedBlockHeader
-	msgToSend := consensus.constructAnnounceMessage()
+	key, err := consensus.GetConsensusLeaderPrivateKey()
```

Comment:
This call is a bit expensive. Why can't we store the key in a field and directly read it? It's doing the looping everytime.

Reply:
Agreed. I will add this optimization to the open staking multi-key support.